### PR TITLE
Chachacha deployment

### PR DESCRIPTION
**Pull request by c410-f3r**
Superseeds #69 

WIP

### CODE REVIEW

File: .github/workflows/chachacha.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        uses: actions/checkout@v2
+
+      - name: Publish
+        uses: appleboy/ssh-action@master
+        with:
+          host: 45.33.117.205
+          key: ${{ secrets.CHACHACHA_VM_PRIVATE_KEY }}
+          passphrase: ${{ secrets.CHACHACHA_VM_PASSWORD }}
+          script: ./.github/workflows/chachacha.sh
+          username: zeitgeist
```
Comment (lsaether) [inline .github/workflows/chachacha.yml:25]:
```suggestion
          username: ci
```

**Comment by c410-f3r** [comment]
CI has a hard limit of 1 hour and Docker parachain image deployment took ~58 minutes. That said, future intermittent failures won't be a surprise.

**Comment by lsaether** [comment]
Are you sure it's a hard limit of 1 hour? [The docs](https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration) says 6 hours is the execution limit of a single job

**Comment by c410-f3r** [comment]
@lsaether You are right. My reasoning was based https://github.com/zeitgeistpm/zeitgeist/actions/runs/714515125 and https://github.com/zeitgeistpm/zeitgeist/actions/runs/714616785 but they failed due to other problems.

**Comment by c410-f3r** [comment]
Ok, everything is now properly set-up. With automatic deployment of docker images and parachains we will be able to quickly test and respond to the latest upstream changes. For mainnet though, a more robust, manageable and scalable strategy will likely be needed.

@lsaether To save CI times, deployments were configured to respond to manual actions, i.e., a click of a button.

**Review by lsaether — APPROVED** [review]

**Pull request merged**